XenAdmin/Wizards/CrossPoolMigrateWizard/Filters/CrossPoolMigrateCanMigrateFilter.cs

```diff
@@ -91,6 +117,16 @@ public override bool FailureFound
                                 continue;
                             }
 
+                            if (vmCache.ContainsKey(host.opaque_ref))
+                            {
+                                disableReason = vmCache[host.opaque_ref];
+                                if (!disableReason.Equals(string.Empty) && !excludedHosts.Contains(host.opaque_ref))

```

Comment:
Apologies for not spotting this earlier, but our preferred method for checking for empty strings is using the function `string.IsNullOrEmpty()`, this way covering for null values as well.

Reply:
Good comment, I forgot about IsNullOrEmpty(). We should use this.